src/status_im/config.cljs

```diff
@@ -36,6 +37,7 @@
 (def goerli-chain-explorer-link "https://goerli.etherscan.io/address/")
 (def optimism-goerli-chain-explorer-link "https://goerli-optimistic.etherscan.io/address/")
 (def opensea-api-key OPENSEA_API_KEY)
+(def wallet-connect-project-id WALLET_CONNECT_PROJECT_ID)
```

Comment:
Not sure why we do this but we can use the `WALLET_CONNECT_PROJECT_ID` directly in other places.